packages/fr_FR/French.dic

```diff
@@ -69421,6 +69421,7 @@ sari/S.
 sarigue/S.
 sarin/S.
 sarisse/S.
+Sàrl
```

Comment:
What does `Sàrl` mean? I'm not finding it in my French dictionary.

Reply:
It's mostly used in switzerland country for "Société à responsabilité limitée".
see here https://www.larousse.fr/dictionnaires/francais/S_A_R_L_/71029 for more info